osc-staging.py

```diff
@@ -395,9 +395,14 @@ def do_staging(self, subcmd, opts, *args):
             CheckDuplicateBinariesCommand(api).perform(opts.save)
         elif cmd == 'freeze':
             for prj in args[1:]:
-                prj = api.prj_from_letter(prj)
+                is_adi = False
+                if 'adi:' in prj:
+                    prj = api.adi_prj_from_adi_name(prj)
+                    is_adi = True
+                else:
+                    prj = api.prj_from_letter(prj)
```

Comment:
`api.prj_from_short()` should handle both cases and provide the full project name.